deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
It's because this is the shim, the Core is brought in as a dep, but this is closely linked to node.